dev-tools/ci-integration/step1-changed_files.sh

```diff
@@ -0,0 +1 @@
+IFS=$'\n'; CHANGED_FILES=($(git diff --name-only --diff-filter=ACMRTUXB "${COMMIT_RANGE}")); unset IFS
```

Comment:
Splitting the script into single-line files looks overkill to me. Wouldn't it be easier to have a single file instead?